### DISCUSSION TITLE
High-pass filter on fail screen is applied even after the fail sound stopped playing

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#15225] [Q&A] [answered]
If you fail a song, and then decide to open the beatmap downloader with ctrl-d, the music previews will sound muffled due to the fail screens new high pass filter, even after the fail sound itself stopped playing.

how to repro:
1) fail a song
2) press ctrl-d
3) click the preview of any song
4) ?!?! it's muffled ?!?!

video: https://streamable.com/cyd6r1
log: [runtime.log](https://github.com/ppy/osu/files/7389387/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15228
